sessionctx/variable/session.go

```diff
@@ -1423,6 +1423,9 @@ type SessionVars struct {
 	// If there exists an index whose estimated selectivity is smaller than this threshold, the optimizer won't
 	// use the ExpectedCnt to adjust the estimated row count for index scan.
 	OptOrderingIdxSelThresh float64
+
+	// OptimizerFixControl control some details of the optimizer behavior through the tidb_opt_fix_control variable.
```

Comment:
Please give the example of this session variables and explain what is this session variables used for

Reply:
The expected usage is in the issue. You can have a look.